ui/src/chat/ChatInput/ChatInput.tsx

```diff
@@ -80,14 +83,20 @@ export default function ChatInput({
   showReply = false,
   sendDisabled = false,
   sendMessage,
+  inThread = false,
 }: ChatInputProps) {
   const [uploadError, setUploadError] = useState<string | null>(null);
+  const [searchParams, setSearchParams] = useSearchParams();
+  const chatReplyId = searchParams.get('chat_reply');
+  const [replyCite, setReplyCite] = useState<{ cite: Cite }>();
+  const groupFlag = useGroupFlag();
   const subscription = useSubscriptionStatus();
   const pact = usePact(whom);
   const id = replying ? `${whom}-${replying}` : whom;
   const chatInfo = useChatInfo(id);
-  const reply = replying || chatInfo?.replying || null;
-  const replyingWrit = reply && pact.writs.get(pact.index[reply]);
+  // const reply = replying || chatInfo?.replying || null;
+  // const replyingWrit = reply && pact.writs.get(pact.index[reply]);
+  const replyingWrit = chatReplyId && pact.writs.get(pact.index[chatReplyId]);
```

Comment:
Unclear why we needed to replace the chat store behavior with routing. It seems the same?